tests/nemo_text_processing/it/data_text_normalization/test_cases_whitelist.txt

```diff
@@ -1,5 +1,3 @@
-Avv.~avvocato
```

Comment:
Whhy deletion?

Reply:
The case passes `pyTests` but fails with `Sparrowhawk`.